src/Functions/serial.cpp

```diff
@@ -0,0 +1,161 @@
+#include <Common/ZooKeeper/ZooKeeper.h>
```

Comment:
`serial` is unrelated to `generateSnowflakeID`. Feel free to split it into a new PR. This will make things easier to roll back in case there is a problem.